File: core/src/ru/myitschool/dcrawler/math/MathAction.java
package ru.myitschool.dcrawler.math;

/**
 * Created by Voyager on 04.08.2017.
 */
public abstract class MathAction {

    public abstract int act();

    public abstract String getDescription();

    public int max(){
        return act();
    }

    public int min(){
        return act();
    }
}

File: core/src/com/wolg_vlad/dcrawler/math/MathAction.kt
package com.wolg_vlad.dcrawler.math

/**
 * Created by Voyager on 04.08.2017.
 */
abstract class MathAction {
    abstract fun act(): Int
    abstract val description: String?
    open fun max(): Int {
        return act()
    }

    open fun min(): Int {
        return act()
    }
}